.circleci/config.yml

```diff
@@ -6,7 +6,6 @@ jobs:
       - checkout
       - run: cd docs && gem install bundler && bundle install && bundle exec jekyll algolia
       - run: npm install
-      - run: npm run update-assets-docs
```

Comment:
So you've removed the auto-compilation? Why is that?

Reply:
Because that actually does not have any effect. GitHub needs the actual source code to be updated via a commit to trigger the GitHub pages recompilation. Therefore, for the time being we need to run the command locally and commit the updated markdown file.

Reply:
OK. It’s still better than running it from jsfiddle 👍🏼